1_fetch/src/get_nwis_data.R

```diff
@@ -1,48 +1,33 @@
+# Download site data from USGS site file site
+# Save output as CSV
+download_nwis_site_data <- function(site_num, fileout, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
 
-download_nwis_data <- function(site_nums = c("01427207", "01432160", "01436690", "01466500")){
-  
-  # create the file names that are needed for download_nwis_site_data
-  # tempdir() creates a temporary directory that is wiped out when you start a new R session; 
-  # replace tempdir() with "1_fetch/out" or another desired folder if you want to retain the download
-  download_files <- file.path(tempdir(), paste0('nwis_', site_nums, '_data.csv'))
-  data_out <- data.frame()
-  # loop through files to download 
-  for (download_file in download_files){
-    download_nwis_site_data(download_file, parameterCd = '00010')
-    # read the downloaded data and append it to the existing data.frame
-    these_data <- read_csv(download_file, col_types = 'ccTdcc')
-    data_out <- bind_rows(data_out, these_data)
-  }
-  return(data_out)
-}
-
-nwis_site_info <- function(fileout, site_data){
-  site_no <- unique(site_data$site_no)
-  site_info <- dataRetrieval::readNWISsite(site_no)
-  write_csv(site_info, fileout)
-  return(fileout)
-}
-
-
-download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
-  
-  # filepaths look something like directory/nwis_01432160_data.csv,
-  # remove the directory with basename() and extract the 01432160 with the regular expression match
-  site_num <- basename(filepath) %>% 
-    stringr::str_extract(pattern = "(?:[0-9]+)")
-  
   # readNWISdata is from the dataRetrieval package
   data_out <- readNWISdata(sites=site_num, service="iv", 
                            parameterCd = parameterCd, startDate = startDate, endDate = endDate)
 
-  # -- simulating a failure-prone web-sevice here, do not edit --
+  # -- simulating a failure-prone web-service here, do not edit --
   set.seed(Sys.time())
   if (sample(c(T,F,F,F), 1)){
     stop(site_num, ' has failed due to connection timeout. Try tar_make() again')
   }
   # -- end of do-not-edit block
   
-  write_csv(data_out, file = filepath)
-  return(filepath)
+  write_csv(data_out, file = fileout)
+  return(fileout)
+
+}
+
+# Read download CSVs and bind them into a single data frame
+combine_csvs <- function(csv){
```

Comment:
An argument called `csv` is a bit ambiguous - could be the files _input_ or _output_ from the function. I would suggest matching the arg naming pattern you have for output filepaths and going with `filesin`, `files_in`,  or something along those lines.